### PR TITLE
Walletconnect fixes

### PR DESCRIPTION
**Pull request by xPi2**

### CODE REVIEW

File: packages/web-app/hooks/useAgreement.tsx [hunk 1]
```diff
@@ -77,6 +77,11 @@ export const useAgreementCreate = ({
 			}
 		},
 		onSuccess,
+		overrides: {
+			gasLimit: 190000,
+			// maxFeePerGas: 250000000,
+			// maxPriorityFeePerGas: 250000000,
+		},
```
Comment (xPi2) [inline packages/web-app/hooks/useAgreement.tsx:84]: I have adjusted the gas limit for every operation. I was unable to reproduce the error that justified fixing gas fees neither with metamask or walletconnect. I can try again if you give me more context.

Comment (gordo-labs) [reply packages/web-app/hooks/useAgreement.tsx:84]: if it works and each gas limit does enters into conflict, all good!

File: packages/web-app/components/dispute/DisputeActions.tsx [hunk 2]
```diff
@@ -233,12 +233,25 @@ export const DisputeActions = () => {
...
+						<a
+							className="underline"
+							href="https://docs.nation3.org/agreements/submitting-evidence"
+							target="_blank"
+							rel="noreferrer"
+						>
+							Submit process
+						</a>
+					</p>
+				</div>
```
[the remark this answers is not in the document]
Comment (gordo-labs) [reply packages/web-app/components/dispute/DisputeActions.tsx:254]: To remove the CTA button, strange behaviour for a CTA in that place.

[the remark this answers is not in the document]
Comment (gordo-labs) [reply packages/web-app/components/dispute/DisputeActions.tsx:254]: To remove the CTA button, strange behaviour for a CTA in that place.

**Review by gordo-labs — APPROVED** [review]

**Pull request merged**